e2e/integration/feedbacktests.js

```diff
@@ -3,19 +3,19 @@ context( 'Test feedback form', () => {
 
     before( () => {
         cy.viewport(Cypress.env('width'), Cypress.env('height') );
-        cy.visit( Cypress.env( 'QA_HOST' ),{timeout:10000} );
-        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env( 'USERNAME' ) );
+        cy.visit( Cypress.env( 'QA_HOST' ) +'feedback');
+        cy.get( '#username' ).type( Cypress.env( 'USERNAME' ) );
         cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
-        cy.get( '#submit',{timeout:10000} ).click();
+        cy.get( '#submit').click();
     } );
 
     it( 'Report bug', () => {
         cy.get( '#feedback-popup', { timeout: 10000 } ).click().wait( 1000 );
         cy.get( `[data-feedback-type="bug"]`, { timeout: 10000 } ).click();
```

Comment:
Do we still need the timeout here?

Reply:
fixed @hybridx https://github.com/1-Platform/one-platform/pull/1484/commits/cca93f81599192904ea85ae0e4b94a12495eb660

Reply:
If this is fixed can you please review and merge

Reply:
@hybridx